### PR TITLE
Update pathgo-edit.owl

### PR DESCRIPTION
**Pull request by kzudock**
Added definitions for PATHGO 200-204 and edited definitions for 248 and 322 (also in that host cell endomembrane dynamics modulation factor grouping) to make them fit the standard definition format since I noticed they did not. Addresses part of issue #142
Added definitions to PATHGO 237-241. Addressed second half of issue #142

### CODE REVIEW

File: src/ontology/pathgo-edit.owl [hunk 1]
```diff
@@ -1753,6 +1755,7 @@ build out subclasses</skos:editorialNote>
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000202">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000199"/>
         <rdfs:label>phagosome acidification disruption factor</rdfs:label>
+        <skos:definition>A gene product that modulates host cell endomembrane dynamics by dirupting phagosome acidification.</skos:definition>
```
Comment (jproesch) [inline src/ontology/pathgo-edit.owl:1758]: fix spelling error- disrupting

File: src/ontology/pathgo-edit.owl [hunk 2]
```diff
@@ -1762,6 +1765,7 @@ build out subclasses</skos:editorialNote>
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000203">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000199"/>
         <rdfs:label>host cell phagosome maturation disruption factor</rdfs:label>
+        <skos:definition>A gene product that modulates host cell endomembrane dynamics by dirupting host cell phagosome maturation.</skos:definition>
```
Comment (jproesch) [inline src/ontology/pathgo-edit.owl:1768]: fix spelling error- disrupting

File: src/ontology/pathgo-edit.owl [hunk 3]
```diff
@@ -1771,6 +1775,7 @@ build out subclasses</skos:editorialNote>
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000204">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/pathgo/PATHGO_0000199"/>
         <rdfs:label>phagolysosome fusion disruption factor</rdfs:label>
+        <skos:definition>A gene product that modulates host cell endomembrane dynamics by dirupting phagosome fusion.</skos:definition>
```
Comment (jproesch) [inline src/ontology/pathgo-edit.owl:1778]: fix spelling error- disrupting
change phagosome to phagolysosome

**Review by jproesch — CHANGES_REQUESTED** [review]

File: src/ontology/pathgo-edit.owl (hunk 1 above)
Comment (kzudock) [reply src/ontology/pathgo-edit.owl]: fixed!

File: src/ontology/pathgo-edit.owl (hunk 2 above)
Comment (kzudock) [reply src/ontology/pathgo-edit.owl]: fixed!

File: src/ontology/pathgo-edit.owl (hunk 3 above)
Comment (kzudock) [reply src/ontology/pathgo-edit.owl]: fixed!

**Review by jproesch — APPROVED** [review]

**Pull request merged**